README.md

```diff
@@ -92,6 +92,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Padloc](https://github.com/padloc/padloc) - Modern, open source password manager for individuals and teams.
 - [Ghorbu Wallet](https://github.com/matthias-wright/ghorbu-wallet) - Cross-platform desktop HD wallet for Bitcoin.
 - [Stable Diffusion Buddy](https://github.com/breadthe/sd-buddy) - Desktop UI companion for the self-hosted Mac version of Stable Diffusion.
+- [Clipborad Record](https://github.com/lesterhnu/clipboard) - Record Clipboard Content, using react-ts+sqlite+tauri
```

Comment:
You should omit the references to technologies used, they might change down the road and offer no benefit to the reader.

```suggestion
- [Clipborad Record](https://github.com/lesterhnu/clipboard) - Record Clipboard Content
```

But if you insist on having the packages in the description please wrap them in backticks.

```suggestion
- [Clipborad Record](https://github.com/lesterhnu/clipboard) - Record Clipboard Content, using `react-ts`+`sqlite`+`tauri`
```